ocaml/typing/primitive.mli

```diff
@@ -27,7 +27,7 @@ type boxed_vector = Pvec128 of vec128_type
    of a primitive *)
 type native_repr =
   | Repr_poly
-  | Same_as_ocaml_repr of Jkind.Sort.const
+  | Same_as_ocaml_repr of Jkind_types.Sort.const
```

Comment:
But the comment in jkind_types.mli says not to depend on it. I understand why you did, here, but maybe that comment should be updated to list the modules that sit in the dependency chain between jkind.ml and jkind_types.ml.